.gitignore

```diff
@@ -48,4 +48,6 @@ test.conf
 spark-warehouse
 *.flattened-pom.xml
 
-seatunnel-examples
\ No newline at end of file
+seatunnel-examples
+
+connectors
```

Comment:
```suggestion
```

Reply:
@Hisoka-X  Removed.
The reason I added it is because when i run example locally it will try to search to this path. is there something I misconfigured?